hack/create-s3-bucket.sh

```diff
@@ -6,29 +6,32 @@ BUCKET_NAME=$1
 
 REGION=$(aws configure get region)
 
-create_bucket() {
+recreate_bucket() {
```

Comment:
@Red-GV this is exactly what I am afraid of and want to solve. I want every run to work from clean new deployment including the bucket. This might be less significant for the write path but in the read path Loki might behave differently is there is "too much data" so old data might affect the results. This assures that we work from same clean env always.

Reply:
The problem with this is that this does not work for the multi-configuration scenario. The same bucket will always be used when doing it in that fashion. We can have a delete script to blow up the bucket expressly. However, I don't agree that - in this case - this script should be blowing up the bucket and then creating a new one.

Actually using one bucket, in that sense, is slightly better. The read path from the client is configured to look through a specific range (https://github.com/ViaQ/cluster-logging-load-client/blob/main/loadclient/query.go#L61), so my expectation is that this scenario of "too much old data" does not occur within the read test.

Reply:
Fixed